### PR TITLE
ci: split secrets and environment variables

### LINKED ISSUE #311: Split environment variables from secrets
Secrets and environment variables are currently all mixed under Github secrets. We can use the corresponding "variables" feature to free up some secrets slots (https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository)

- [x] Replace in the caller workflows
- [ ] Replace in graasp-deploy
- [ ] Replace the "secrets" into variables in the repo / org settings

⚠️ make sure to not leak actual secrets while doing this

### PR DESCRIPTION
**Pull request by dialexo**
Closes #311

Not sure about the following "secrets" - should they be moved to variables also? @spaenleh @pyphilia 
- `aws-region` ✅ 
- `ecr-repository` ❌ 
- `ecs-cluster` ✅ 
- `ecs-service` ✅ 
- `h5p-content-bucket-name` ❌ 
- `h5p-content-region` ✅ 
- `redis-host` ❌ 
- `redis-port` ❌ 
- `s3-file-item-bucket` ❌ 
- `s3-file-item-region` ✅

### CODE REVIEW

File: .github/workflows/deploy-development.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# This workflow is made up of one job that calls the reusable workflow in graasp-deploy
+jobs:
+  graasp-deploy-ecs-backend-workflow:
+    # repository name
+    name: Graasp
+    uses: graasp/graasp-deploy/.github/workflows/cintegration-ecs-backend.yml@v1
```
Comment (spaenleh) [inline .github/workflows/deploy-development.yml:19]: Do you also plan to rename the files in deploy ?

File: .github/workflows/deploy-development.yml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    with:
+      # workflow config
+      ecs-task-definition: '.aws/graasp-dev.json'
+      # env vars
+      analyzer-client-host: ${{ vars.ANALYZER_CLIENT_HOST }}
+      apps-plugin: true
+      auth-client-host: ${{ vars.AUTH_CLIENT_HOST }}
+      auth-token-expiration-in-minutes: ${{ vars.AUTH_TOKEN_EXPIRATION_IN_MINUTES }}
+      avatars-path-prefix: ${{ vars.AVATARS_PATH_PREFIX }}
+      aws-region: ${{ secrets.AWS_REGION }}
```
Comment (spaenleh) [inline .github/workflows/deploy-development.yml:30]: Move this to vars ?

File: .github/workflows/deploy-development.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      container-image-classifier: ${{ vars.CONTAINER_IMAGE_CLASSIFIER }}
+      container-image-iframely: ${{ vars.CONTAINER_IMAGE_IFRAMELY }}
+      container-name-classifier: ${{ vars.CONTAINER_NAME_CLASSIFIER }}
+      container-name-graasp: ${{ vars.CONTAINER_NAME_GRAASP }}
+      container-name-iframely: ${{ vars.CONTAINER_NAME_IFRAMELY }}
+      cookie-domain: ${{ vars.COOKIE_DOMAIN }}
+      cors-origin-regex: ${{ vars.CORS_ORIGIN_REGEX }}
+      database-logs: false
+      ecs-cluster: ${{ secrets.ECS_CLUSTER_GRAASP }}
+      ecs-service: ${{ secrets.ECS_SERVICE_GRAASP }}
```
Comment (spaenleh) [inline .github/workflows/deploy-development.yml:43]: Same question, move this to vars ?

File: .github/workflows/deploy-development.yml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      image-classifier-api: ${{ vars.IMAGE_CLASSIFIER_API }}
+      node-env-iframely: production
+      node-env: production
+      player-client-host: ${{ vars.PLAYER_CLIENT_HOST }}
+      port: ${{ vars.PORT }}
+      public-plugin: true
+      refresh-token-expiration-in-minutes: ${{ vars.REFRESH_TOKEN_EXPIRATION_IN_MINUTES }}
+      roarr-log: false
+      s3-file-item-plugin: true
+      s3-file-item-region: ${{ secrets.S3_FILE_ITEM_REGION }}
```
Comment (spaenleh) [inline .github/workflows/deploy-development.yml:65]: Move to vars ?

**Review by spaenleh — COMMENTED** [review]
Looks good, just wondering If you wanted to move some secrets to vars as we discussed yesterday.

**Comment by dialexo** [comment]
> Looks good, just wondering If you wanted to move some secrets to vars as we discussed yesterday.

Good catch, I moved some to inputs but forgot to use the vars namespace... oops 😅

File: .github/workflows/deploy-development.yml (hunk 1 above)
Comment (dialexo) [reply .github/workflows/deploy-development.yml:19]: At some point yes. But I think i'll refactor with the Github environments also when doing so.

**Review by spaenleh — APPROVED** [review]
All Good ! 🐭

**Pull request merged**